.github/workflows/examples_e2e.yml

```
@@ -0,0 +1,43 @@
+name: Examples / E2E
+
+on:
+  workflow_call:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - 'examples/**'
+
+concurrency:
+  group: examples-e2e-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test_web:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        example_dir: ['examples/web-wa-sqlite', 'examples/web-wa-sqlite-vuejs']
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+      - name: Install dependencies
+        working-directory: ${{ matrix.example_dir }}
+        run: npm ci
+      - name: Start backend, run migrations, generate client
+        working-directory: ${{ matrix.example_dir }}
+        run: |
+          npm run backend:up &&
+          npm run db:migrate &&
+          npm run client:generate
+      - name: Start dev server in background
+        working-directory: ${{ matrix.example_dir }}
+        run: npm run dev < /dev/null &
+      - name: Run e2e tests
+        working-directory: ${{ matrix.example_dir }}
+        run: npm run e2e
```

Slightly more consise

```suggestion
    defaults:
      run:
        working-directory: ${{ matrix.example_dir }}
    steps:
      - uses: actions/checkout@v4
      - uses: actions/setup-node@v4
        with:
          node-version: lts/*
      - name: Install dependencies
        run: npm ci
      - name: Start backend, run migrations, generate client
        run: |
          npm run backend:up &&
          npm run db:migrate &&
          npm run client:generate
      - name: Start dev server in background
        run: npm run dev < /dev/null &
      - name: Run e2e tests
        run: npm run e2e
```

---

Ah thank you, I was looking for that config!